scandipwa/cms-graphql
# Changed the scope of the di.xml to graphql from global

Fixes issue #8 - CMS widgets in Luma theme breaks after installing scandipwa/cms-graphql

## Manual testing scenarios
1. Create any widget in a CMS page which uses `ScandiPWA\CmsGraphQl\Model\Template\VirtualFilter::availableFilters` (for eg, Slider, NewProducts, or, CatalogProductList)
2. Open the CMS page in Luma theme (NOT the pwa theme).
3. __Behaviour before PR__ The widget will be rendered on HTML as `<widget type="...` because of which it is not rendered on the page
4. __Behaviour after PR__ The widget HTML is rendered as expected using the respective widget's block.
